### PR TITLE
Feature/tables

### PR DESCRIPTION
**Pull request by bengammon**
Issue: https://github.com/EtchUK/Etch.OrchardCore.ThemeBoilerplate/issues/87

Add basic table styles and modifier classes to the boilerplate.

### CODE REVIEW

**Comment by bengammon** [comment]
@peterkeating @Newnab I have added useful notes to the component

File: content/Assets/Styles/base/_table.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            @include rem(padding-bottom, $table-cell-padding-vertical);
+            padding-right: 0;
+            @include rem(padding-top, $table-cell-padding-vertical);
+            vertical-align: top;
+
+            text-align: left;
+        }
+
+        &:last-child {
+            @include rem(padding-right, $table-cell-padding-horizontal);
```
Comment (Newnab) [inline content/Assets/Styles/base/_table.scss]: Is it just me or is this line redundant? Or should this last child bit be nested inside the bp-min-width mixin above?

Comment (peterkeating) [reply content/Assets/Styles/base/_table.scss]: Agreed, looks redundant as the default `td` has the same padding.

Comment (peterkeating) [reply content/Assets/Styles/base/_table.scss]: Tell a lie, it is needed otherwise the last column buts up right against the edge of the table.

**Pull request merged**